tools/export_slow_tests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+from tools.stats_utils.s3_stat_parser import get_previous_reports_for_branch, Report, Version2Report
+from typing import cast, Dict, Tuple, List
+
+SLOW_TESTS_FILE = '.pytorch-slow-tests'
+SLOW_TEST_CASE_THRESHOLD = 60.0
+
+
+def get_test_case_times() -> Dict[str, float]:
+    # an entry will be like ("test_doc_examples (__main__.TestTypeHints)" -> (current_avg, # values))
+    reports: List[Report] = get_previous_reports_for_branch('origin/viable/strict', "")
+    test_names_to_times: Dict[str, Tuple[float, int]] = dict()
+    for report in reports:
+        if 'format_version' not in report:  # version 1 implicitly
```

Comment:
I'd suggest explicitly checking for version 2, rather than just checking that it's _not_ version 1:
```suggestion
        if report.get('format_version') == 2:
```

Reply:
probably also need a default in case format_version doesn't exist?

Reply:
Nit: can we add a default value, i.e. report.get('format_version', 1) < 2:

Reply:
@walterddr no, that's handled by `.get`

Reply:
@malfet that seems unnecessarily complicated, and wouldn't work properly if other versions are added besides version 2 (which is what my suggestion was meant to avoid)